create-db-script/check-create-pgdb.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
+# ----------Variable Section-----------#
```

Comment:
Copyright header required - see [here](https://github.com/CAFapi/opensuse-tomcat-image/blob/v3.0.0/src/main/docker/startup.d/setup-tomcat-ssl-cert.sh) for example.